docs/develop/python/message-passing.mdx

```diff
@@ -572,7 +572,7 @@ When working with Updates, in addition to the `RPCError`s described above, you m
 
       - A failed Child Workflow
       - A failed Activity (if the Activity retries have been set to a finite number)
-      - The Workflow author raising `ApplicationFailure`
+      - The Workflow author raising [`ApplicationError`](/references/failures#application-failure)
```

Comment:
Looked this up by following the link to confirm mismatch between exception/error type and anchor. Seems fine.

TypeScript: [ApplicationFailure](https://typescript.temporal.io/api/classes/common.ApplicationFailure)
Java: [ApplicationFailure](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/failure/ApplicationFailure.html)
Go: [ApplicationError](https://pkg.go.dev/go.temporal.io/sdk/temporal#ApplicationError)
Python: [ApplicationError](https://python.temporal.io/temporalio.exceptions.ApplicationError.html)
Proto: [ApplicationFailureInfo](https://api-docs.temporal.io/#temporal.api.failure.v1.ApplicationFailureInfo) and [Failure](https://api-docs.temporal.io/#temporal.api.failure.v1.Failure)